examples/oit_reconstruction_new.py

```diff
@@ -0,0 +1,981 @@
+# Copyright 2014-2016 The ODL development group
```

Comment:
To elaborate further on what @ozanoktem said:

The content of a pull request should be things that you expect others to read and give feedback on. Either fully finished code that you feel is ready for a review and merge, or partially finished code that you want someone else to have a look at and provide input on. For example, you may want input on if a design is good before you make further work on it.

Another guideline is that all code in a pull request should be runnable *as is* so that others can try your code on their own system and verify that it works. Thus you should not have any dependencies to external code that is not listed in the official dependency list (basically `numpy`, `scipy` and `matplotlib`).